### ISSUE TITLE
Accessibility help is not accessible when Screen Reader Accessibility Mode is off

### ISSUE THREAD

**Issue by rperez030** [microsoft/vscode#188278]
Type: <b>Bug</b>

When accessibility mode is off and focus is in the editor, pressing Alt +F1 produces the following announcement to screen reader users: "The editor is not accessible at this time. Press Alt+F1 for options.". If one is a new user of the product and screen reader mode doesn't turn on automatically, pressing the command which is supposed to help one get started just repeats the same announcement inviting the user to use Alt +F1 to obtain help. @meganrogge  @isidorn 

VS Code version: Code - OSS Dev 1.81.0 (Commit unknown, Date unknown)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz (16 x 2793)|
|GPU Status|2d_canvas: unavailable_software<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software<br>webgpu: unavailable_software|
|Load (avg)|undefined|
|Memory (System)|63.95GB (48.76GB free)|
|Process Argv|.|
|Screen Reader|yes|
|VM|0%|
</details>

**Comment by meganrogge** [comment]
Since the help menu is not accessible with that off, instead of `use alt+f1 for help` say `Run the command Toggle Screen Reader Accessibility Mode` to enable screen reader mode.
